openedx_webhooks/github_views.py

```diff
@@ -33,10 +33,13 @@ def pull_request():
 
     .. _PullRequestEvent: https://developer.github.com/v3/activity/events/types/#pullrequestevent
     """
+    # TODO: We need to untangle this, there are **four** `return`s in this function!
     try:
         event = request.get_json()
     except ValueError:
-        raise ValueError("Invalid JSON from Github: {data}".format(data=request.data))
+        msg = "Invalid JSON from Github: {data}".format(data=request.data)
+        print(msg, file=sys.stderr)
+        raise ValueError(msg)
     sentry.client.extra_context({"event": event})
 
```

Comment:
I think it would be useful right here to add a log line that an event was received.  It will help us diagnose problems if something goes wrong before one of the exiting log lines is reached.